src/vhdl/vhdl-sem_psl.adb

```diff
@@ -123,6 +123,41 @@ package body Vhdl.Sem_Psl is
       return Call;
    end Sem_Prev_Builtin;
 
+   function Sem_Stable_Builtin (Call : Iir; Atype : Iir) return Iir
+   is
+      use Vhdl.Sem_Expr;
+      use Vhdl.Std_Package;
+      Expr  : Iir;
+      Clock : Iir;
+      First : Boolean;
+   begin
+      Expr := Get_Expression (Call);
+      First := Is_Expr_Not_Analyzed (Expr);
+      Expr := Sem_Expression_Ov (Expr, Atype);
```

Comment:
Instead of calling `Sem_Expression_Ov`, call `Sem_Expression` with `Null_Iir` as type.